sdk/datamigration/ci.mgmt.yml

```diff
@@ -0,0 +1,13 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
We should also use the script to update `eng\scripts Update-Mgmt-CI.ps1` to add these new RPs paths in Azure.ResourceManager ci.mgmt.yml file.

Reply:
done